.github/workflows/csharp-qltest.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           key: csharp-qltest-${{ matrix.slice }}
       - name: Run QL tests
         run: |
-          codeql test run --threads=0 --ram 50000 --slice ${{ matrix.slice }} --search-path extractor-pack --check-databases --check-undefined-labels --check-repeated-labels --check-redefined-labels --consistency-queries ql/consistency-queries ql/test --compilation-cache "${{ steps.query-cache.outputs.cache-dir }}"
+          codeql test run --threads=0 --ram 50000 --slice ${{ matrix.slice }} --search-path "${{ github.workspace }}" --check-databases --check-undefined-labels --check-repeated-labels --check-redefined-labels --consistency-queries ql/consistency-queries ql/test --compilation-cache "${{ steps.query-cache.outputs.cache-dir }}"
```

Comment:
I wonder, does `codeql` fail if you provide a non-existing `--search-path`? if it's the case, there is a change of behaviour to consider here, as this command would previously fail if for some reason the extractor-pack wasn't created, while it would silently pick up the released pack now. Just a corner case to consider, I'm also happy to keep this like this for the sake of simplification.

Reply:
> I wonder, does codeql fail if you provide a non-existing --search-path?

No it doesn't, so the corner case you mention already exists.

Reply:
At least `gh codeql resolve languages --search-path blabla` just "works" without complaints.